### PR TITLE
Add IntelliJ run steps

### PR DESCRIPTION
**Pull request by gidsg**
...as this did not work out of the box.

### CODE REVIEW

**Comment by karlbaker02** [comment]
This is cool. Just a few small comments I'll put in the appropriate places now...

File: README.md [hunk 1]
```diff
@@ -50,6 +50,22 @@ If the basic registers change you can restart your register so that it sees the
 - Run the `./go` script to set everything up
 - Build project using command `./gradlew clean build`
 
+### IntelliJ setup
+- `File > Open` your checkout directory
+- `Run > Edit Configurations`
+- Edit `RegisterRunner`
+- Use classpath of module: `openregister-java_main`
+- JRE: `1.8`
```
Comment (karlbaker02) [inline README.md]: This will be whichever JRE the user is running, although probably 1.8

File: README.md [hunk 2]
```diff
@@ -50,6 +50,22 @@ If the basic registers change you can restart your register so that it sees the
...
+- `Run > Run... > RegisterRunner`
+
+To test if everything has worked:
+
+`$ curl 127.0.0.1:8080/records -H "Host: field"`
```
Comment (karlbaker02) [inline README.md]: I'd maybe change the host header here to be `-H "Host: school"` as this is the register which will run on port `8080` when the register starts up

Comment (gidsg) [inline README.md]: When I do this it still returns `Register undefined` is that expected?

Comment (karlbaker02) [inline README.md]: Yep, that's expected.

**Pull request merged**